crypto/sm2/sm2_pmeth.c

```diff
@@ -1,391 +0,0 @@
-/*
```

Comment:
There should be more of this sort that we can drop at this point...

Reply:
... I mean, for other key types

Reply:
Yes - that occurred to me too, but I thought it was outside the scope of this PR to address it.

Reply:
Ah, yes, I didn't mean that it should be done in _this_ PR...  I just noted that it seems to be time to pull that plug.